createFilename.m

```diff
@@ -92,47 +92,38 @@
 [~, ~, ~] = mkdir(expParameters.outputDir);
 [~, ~, ~] = mkdir(fullfile(expParameters.outputDir, modality));
 
-if isfield(cfg,'eyeTracker')
+if cfg.eyeTracker
     [~, ~, ~] = mkdir(fullfile(expParameters.outputDir, 'eyetracker'));
 end
 
 
 %% create filenames
 
 switch modality
-    
+
     case 'beh'
-        
+
         expParameters.fileName.events = ...
             [expParameters.fileName.base, runSuffix, '_events_date-' expParameters.date '.tsv'];
-        
-        expParameters.fileName.stim = ...
-            [expParameters.fileName.base, runSuffix, '_stim_date-' expParameters.date '.tsv'];
-        
+
     case 'func'
-        
+
         expParameters.fileName.events = ...
             [expParameters.fileName.base, ...
             expParameters.acqSuffix, expParameters.ceSuffix, ...
             expParameters.dirSuffix, expParameters.recSuffix, ...
             runSuffix, expParameters.echoSuffix, ...
             '_events_date-' expParameters.date '.tsv'];
-        
-        expParameters.fileName.stim = ...
-            [expParameters.fileName.base, ...
-            expParameters.acqSuffix, expParameters.ceSuffix, ...
-            expParameters.dirSuffix, expParameters.recSuffix, ...
-            runSuffix, expParameters.echoSuffix, ...
-            '_stim_date-' expParameters.date '.tsv'];
```

Comment:
there we go! finally I made it more clear what's missing from Remi's latest changes.

Reply:
I think I am lost :)

Reply:
what happened?

Reply:
The red ones are somehow doesnt exist in your branch. And these are the ones I'll need for my behavioral experiment that Remi implemented just an hour ago. 
Somehow your branch doesn't carry the changes from Remi and I do not want to lose those changes. Not sure how did that happen.

Reply:
ok, I have no idea. Probably some conflict while working in different branches somehow not all at the same stage. I propose that I delete my pr so we don't lose Remi's job. Is it ok?